client/my-sites/customer-home/components/full-screen-launchpad.tsx

```diff
@@ -10,12 +10,14 @@ import useHomeLayoutQuery from 'calypso/data/home/use-home-layout-query';
 import { skipLaunchpad } from 'calypso/landing/stepper/utils/skip-launchpad';
 import { launchSiteApi } from 'calypso/lib/signup/step-actions';
 import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
```

Comment:
Since this requires us to dispatch an event to work, I replaced it with `import { recordTracksEvent } from '@automattic/calypso-analytics'` [here](https://github.com/Automattic/wp-calypso/pull/99891/commits/43125fe15b441083ccd90abaabe8fcd558892daa).